docs/platforms/javascript/common/options.mdx

```diff
@@ -0,0 +1,454 @@
+
+</SdkOption>
+
+<SdkOption name="includeLocalVariables" type='boolean' defaultValue='false'  categorySupported={['server', 'serverless']}>
+
+Set this option to `true` to add stack local variables to stack traces.
+
+For more advanced configuration options, see the documentation on the <PlatformLink to="/configuration/integrations/localvariables">Local Variables integration options</PlatformLink>.
+
+</SdkOption>
+
+<SdkOption name="integrations" type='Array<Integration>' defaultValue='[]'>
+
+Pass additional integrations that should be initialized with the SDK. Integrations are pieces of code that can be used to extend the SDK's functionality. They can be used to add custom event processors, context providers, or to hook into the SDK's lifecycle.
+
+See <PlatformLink to="/configuration/integrations/">integration docs</PlatformLink> for more information.
+
+</SdkOption>
+
+<SdkOption name="defaultIntegrations" type='undefined | false'>
+
+This can be used to disable integrations that are added by default. When set to `false`, no default integrations are added.
+
+See <PlatformLink to="/configuration/integrations/#modifying-default-integrations">integration docs</PlatformLink> to see how you can modify the default integrations.
+
+</SdkOption>
+
+<SdkOption name="transport" type="(transportOptions: TransportOptions) => Transport">
+
+The JavaScript SDK uses a transport to send events to Sentry. On modern browsers, most transports use the browsers' fetch API to send events. Transports will drop an event if it fails to send due to a lack of connection.
+
+<PlatformCategorySection supported={["browser"]}>
+  In the browser, a fetch-based transport is used by default.
+</PlatformCategorySection>
+
+<PlatformCategorySection supported={["server", "serverless"]}>
+  On the server, a `https`-based transport is used by default.
+</PlatformCategorySection>
+
+</SdkOption>
+
+<SdkOption name="transportOptions" type='TransportOptions'>
+
+Options used to configure the transport. This is an object with the following possible optional keys:
+
+<PlatformCategorySection supported={["server", "serverless"]}>
+  - `headers`: An object containing headers to be sent with every request. -
+  `proxy`: A proxy used for outbound requests. Can be http or https. -
+  `caCerts`: A path or list of paths to a CA certificate, or a buffer of CA
+  certificates. - `httpModule`: A custom HTTP module to use for requests.
+  Defaults to the the native `http` and `https` modules. - `keepAlive`:
+  Determines whether to keep the socket alive between requests. Defaults to
+  `false`.
+</PlatformCategorySection>
+
+<PlatformCategorySection supported={["browser", "desktop"]}>
+  - `headers`: An object containing headers to be sent with every request. Used
+  by the SDK's fetch and XHR transports. - `fetchOptions`: An object containing
```

Comment:
are we intentionally not putting the individual transport options into their own lines?

Reply:
no! auto-format messed this up, good catch!